pkg/service/auth_test.go

```
@@ -375,6 +381,172 @@ func TestCheckFailsClosedOnContextTimeout(t *testing.T) {
 	assert.Check(t, denied != nil, "Expected denied response")
 }
 
+func setupTestTracer(t *testing.T) *tracetest.InMemoryExporter {
+	t.Helper()
+	exporter := tracetest.NewInMemoryExporter()
+	tp := sdktrace.NewTracerProvider(sdktrace.WithSyncer(exporter))
+	prev := otel.GetTracerProvider()
+	otel.SetTracerProvider(tp)
+	t.Cleanup(func() {
+		otel.SetTracerProvider(prev)
+		tp.Shutdown(context.Background())
```

golangci-lint (2.12.2)

[error] Error return value of `tp.Shutdown` is not checked

(errcheck)